pentasharp/Migrations/20241104124847_TaxiBookingsAndNotificationsMigration.cs

```diff
@@ -71,7 +71,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
                         column: x => x.UserId,
                         principalTable: "Users",
                         principalColumn: "UserId",
-                        onDelete: ReferentialAction.Cascade);
+                        onDelete: ReferentialAction.NoAction);
```

Comment:
should this be Restrict, similar to this https://github.com/starlabspro-internship/pentasharp/pull/13/files